src/library/styles/stylePrimaryButton.js

```diff
@@ -0,0 +1,46 @@
+import {StyleSheet} from 'react-native';
+
+export const style = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  button: {
+    paddingVertical: 20,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#32B3A5',
+  },
+  text: {
+    fontSize: 24,
+    fontWeight: 'bold',
+    color: 'white',
+  },
+  buttonAddTo: {
+    padding: 5,
+    backgroundColor: 'white',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderWidth: 2,
+    borderColor: '#7C7D87',
+    borderRadius: 8,
+    color: '#7C7D87',
```

Comment:
All the colors, have to be on a constant file and just call it...
Example: {  
    color: colors.dirtyWhite, 
    bacgroundColor: colors.dirtyRed, }